### PR TITLE
Fixes #699 NTR for blood vessels that drain and/or supply

### LINKED ISSUE #699: NTR: supplies blood to; drains blood from
Make new subproperties of supplies/drains: "supplies blood to" / "drains blood from" with domains 'arterial blood vessel' and 'venous blood vessel'' respectively (these classes cover the microvasculature as well as arteries and veins.). 

Background: These will replace usage of supplies and drains in Uberon, where they have been used extensively but with domain and range added locally - illegally restricting the meaning of these relations & being the the potential source of problematic inferences where drains and supplies are used outside of an anatomical context (e.g. for rivers).

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #699 NTR for blood vessels that drain and/or supply.
Also fixes https://github.com/obophenotype/uberon/issues/2831.
Labels have been discussed internally in EBI but suggestions to improve/shorten them are welcome.

### CODE REVIEW

File: src/ontology/ro-edit.owl [hunk 1]
```diff
@@ -6789,6 +6792,37 @@ AnnotationAssertion(oboInOwl:created_by obo:RO_0019501 <https://orcid.org/0000-0
...
+AnnotationAssertion(terms:contributor obo:RO_0020102 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(terms:date obo:RO_0020102 "2024-02-01T10:09:41Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0020102 "vessel drains blood from")
+SubObjectPropertyOf(obo:RO_0020102 obo:RO_0002179)
+ObjectPropertyDomain(obo:RO_0020102 obo:UBERON_0001981)
+ObjectPropertyRange(obo:RO_0020102 obo:UBERON_0000061)
+
+# Object Property: obo:RO_0020103 (vessel directly supplies blood to and drains blood from)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0020103 "x vessel directly supplies blood to and drains blood from y if and only if x is a blood vessel, and it directly supplies blood to y and directly drains blood from y.")
```
Comment (matentzn) [inline src/ontology/ro-edit.owl:6820]: For consistency, maybe say that y is an anatomical entity?

File: src/ontology/ro-edit.owl [hunk 2]
```diff
@@ -6564,15 +6568,14 @@ IrreflexiveObjectProperty(obo:RO_0017004)
 
 # Object Property: obo:RO_0017005 (contains measured amount)
 
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017005 "anticoagulant-containing test tube contains measured amount 5 ml of blood specimen.")
 AnnotationAssertion(obo:IAO_0000115 obo:RO_0017005 "A relation between a container and measurement datum that specifies the actual amount of material in the container.")
 AnnotationAssertion(terms:contributor obo:RO_0017005 <https://orcid.org/0000-0002-2999-0103>)
 AnnotationAssertion(terms:contributor obo:RO_0017005 <https://orcid.org/0000-0002-5714-991X>)
-AnnotationAssertion(obo:IAO_0000112 obo:RO_0017005 "anticoagulant-containing test tube contains measured amount 5 ml of blood specimen.")
```
Comment (matentzn) [inline src/ontology/ro-edit.owl:6570]: @anitacaron do you know why these spurious diffs still exist?

**Review by matentzn — COMMENTED** [review]
I approve this, but would prefer a @cmungall or @balhoff because I am not great at RO modelling. Maybe share with them on slack with not that its urgent.

**Comment by aleixpuigb** [comment]
> I approve this, but would prefer a @cmungall or @balhoff because I am not great at RO modelling. Maybe share with them on slack with not that its urgent.

Thank you for looking into it, @matentzn. I will!

File: src/ontology/ro-edit.owl (hunk 2 above)
Comment (anitacaron) [reply src/ontology/ro-edit.owl:6570]: That's fine. The serialization here sometimes is an issue.

File: src/ontology/ro-edit.owl [hunk 3]
```diff
@@ -47,6 +47,7 @@ Declaration(Class(obo:RO_0002310))
 Declaration(Class(obo:RO_0002532))
 Declaration(Class(obo:RO_0002533))
 Declaration(Class(obo:RO_0002534))
+Declaration(Class(obo:UBERON_0001981))
```
Comment (anitacaron) [inline src/ontology/ro-edit.owl:50]: Please, remove the uberon class definition in the edit file.

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/ro-edit.owl (hunk 1 above)
Comment (aleixpuigb) [reply src/ontology/ro-edit.owl]: Done, thank you

**Review by anitacaron — APPROVED** [review]
Technically, I approve. But it still needs the ontology review.

**Review by balhoff — APPROVED** [review]

**Pull request merged**